README.md

```diff
@@ -37,17 +37,23 @@ The [Atomic Developer Bundle](https://github.com/projectatomic/adb-atomic-develo
   * IRC: #atomic and #nulecule on freenode
   * Mailing List: container-tools@redhat.com
 
-## How to Develop/Test
+## How to Develop/Test?
 
 1. Install the Atomic Developer Bundle (ADB), as [documented](https://github.com/projectatomic/adb-atomic-developer-bundle/blob/master/docs/installing.rst) in the ADB project.  Do not start the box yet.
 
-2. Run `bundle install`
+2. Git clone repo
 
-3. Start the box with `bundle exec vagrant up`
+        git clone https://github.com/projectatomic/vagrant-service-manager
 
-4. Develop.  You can test the command by running `bundle exec vagrant service-manager`
+3. `cd vagrant-service-manager`
 
-5. When you are ready to build the release, get a maintainer to:
+4. Run `bundle install`
+
+5. Start the box with `bundle exec vagrant up`
+
+6. Develop. Test the command by running `bundle exec vagrant service-manager`
```

Comment:
I thought that initially then taken back as it looks like `bundle exec vagrant service-manager` command is used for both develop and test.